### ISSUE TITLE
[Bug]: code and mongocli command not found

### ISSUE THREAD

**Issue by simsketch** [Pythagora-io/gpt-pilot#776]
### Version

VisualStudio Code extension

### Operating System

MacOS

### What happened?

My terminal has access to the node, npm, and mongocli commands, but for some reason when the agents are running, they are not able to find the executable. I am using homebrew for mongocli and nvs for node, but again, both are available in a fresh instance of terminal, so I'm not sure if there's another context they should be running in.

**Comment by senko** [comment]
Can you add screenshot or logs of the errors? You can also check whether these are available from within VSCode (when you open a terminal inside vscode and try to run the commands).

**Comment by simsketch** [comment]
Yes they are available using the VSCode terminal, I just ended up switching to using python in Terminal. I'm on Cursor by the way, I didn't want to keep retrying it because of token usage but I switch to haiku model so I may try again. But I believe it's related to not using the virtual environment?
